roles/ceph-common/tasks/installs/debian_custom_repository.yml

```
@@ -1,7 +1,13 @@
 ---
+- name: configure debian custom apt key
+  apt_key:
+    url: "{{ ceph_custom_key }}"
```

so this becomes a mandatory option when `ceph_custom_repo` is used?

otherwise, we probably need this:

```suggestion
    url: "{{ ceph_custom_key }}"
    when: ceph_custom_key is defined
```

---

OK, I'll go with ` when: ceph_custom_key is defined`